foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -1,18 +1,18 @@
 # Configuration
 
-The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely on Experience Platform Launch or locally.
+The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely in the Data Collection UI or locally.
 
-## Configure with the Experience Platform Launch App ID
+## Configure with the Experience Platform App ID
 
-Experience Platform Launch generates a unique environment ID that the SDK uses to retrieve your configuration. This ID is generated when an app configuration is created and published to a given environment. The app is first launched and then the SDK retrieves and uses this Adobe-hosted configuration.
+Adobe Experience Platform generates a unique environment ID that the SDK uses to retrieve your configuration. This ID is generated when an app configuration is created and published to a given environment. The app is first launched and then the SDK retrieves and uses this Adobe-hosted configuration.
 
 {% hint style="success" %}
-We strongly recommend that you configure the SDK with the Experience Platform Launch environment ID. Follow the steps in [Set up a mobile property](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/50f1aec2d1ab7760583a7ba8dcd97f3c265309f6/getting-started/create-a-mobile-property/README.md) if you need to create a new Experience Platform Launch App.
+We strongly recommend that you configure the SDK with the Experience Platform environment ID. Follow the steps in [Set up a mobile property](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property) if you need to create a new Experience Platform App.
 {% endhint %}
 
 After the configuration is retrieved when the app is initially launched, the configuration is stored in local cache. The SDK tries to refresh the configuration every cold launch or when a new session is detected. If there is no change or a network request error occurs while downloading the configuration file, the cached configuration will be used.
 
-The unique environment ID provided by Experience Platform Launch can be configured with the SDK using the following:
+The unique environment ID provided by Experience Platform can be configured with the SDK using the following:
```

Comment:
Since this is a very subsequent mention, we can just go with "... provided by Platform can be configured..."